### PR TITLE
Editor - Move document menu to document list

### PR DESCRIPTION
**Pull request by jcontonio**
* Moves document menu out of the document header
* Tightens the document list down
* Changes highlight text to yellow

image

### CODE REVIEW

File: web/src/client/js/components/Header/HeaderComponent.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const renderBrandLogo = (logo) => {
 const HeaderComponent = ({ brand, isFullScreen, section, subscriptionStatus }) => {
   const upgradePillClasses = cx({
     'pill': true,
-    'pill--red': LOCKED_ACCOUNT_STATUSES.includes(subscriptionStatus),
+    'pill--red': !LOCKED_ACCOUNT_STATUSES.includes(subscriptionStatus),
     'pill--gray': subscriptionStatus === SUBSCRIPTION_STATUS.TRIALING || subscriptionStatus === null,
```
Comment (dheniges) [inline web/src/client/js/components/Header/HeaderComponent.js]: This is assigning both `pill--gray` and `pill--red`. Is red not supposed to be if it's locked?

**Review by dheniges — COMMENTED** [review]
Everything works that I tried. Thanks for the smaller PR. Only question about what pill class is supposed to be given to what.

File: web/src/client/js/components/Header/HeaderComponent.js (hunk 1 above)
Comment (jcontonio) [reply web/src/client/js/components/Header/HeaderComponent.js]: This is ready for review @dheniges, removed the debugging thing

**Review by dheniges — APPROVED** [review]
Pills look good now

**Pull request merged**